### PR TITLE
merge tree and ls

### PR DESCRIPTION
**Pull request by j-ogas**
Addresses #80.

TODO: 
 - [ ] figure out how to make unit tests

### CODE REVIEW

**Comment by Paul-Ferrell** [comment]
Looking good so far.

File: docs/install.rst [hunk 1]
```diff
@@ -116,7 +118,7 @@ sub-repos. To download them in this manner, simply run:
 
 .. code:: bash
 
-    git submodule update --init
+    git submodule update --init --recursive
```
Comment (j-ogas) [inline docs/install.rst]: Would it be worth mentioning that we can also clone pavilion and the submodules from start with: `clone --recursive https://github.com/hpc/pavilion2`?

Comment (Paul-Ferrell) [reply docs/install.rst]: This is fixed elsewhere. 

I was thinking of putting it in a git post-update hook or something.

**Comment by j-ogas** [comment]
@Paul-Ferrell Not able to reproduce the travis failure on my box.

```
$ ./test/run_tests -o ls
[...]
test_ls (ls_tests.StatusTests)
Checking ls command functionality ... test_ls (ls_tests.StatusTests)
Checking ls command functionality ... Test unittest.quick_test run 103 building fe8e241e30daeb4b
/home/jogas/git/pavilion2/test/working_dir/test_runs/0000103:
config
variables
status
build.sh
build_hash
run.tmpl
build
run.sh
/home/jogas/git/pavilion2/test/working_dir/test_runs/0000103:
config
variables
status
build.sh
build_hash
run.tmpl
build/
    pav_build_log ->
/home/jogas/git/pavilion2/test/working_dir/builds/fe8e241e30daeb4b/pav_build_log
    .built_by -> /home/jogas/git/pavilion2/test/working_dir/test_runs/0000103
run.sh
/home/jogas/git/pavilion2/test/working_dir/test_runs/0000103:
pav_build_log -> /home/jogas/git/pavilion2/test/working_dir/builds/fe8e241e30daeb4b/pav_build_log
.built_by
ok
[...]
----------------------------------------------------------------------
Ran 89 tests in 0.140s

OK (skipped=88)
$ python3 --version
Python 3.7.4
```

**Comment by j-ogas** [comment]
@Paul-Ferrell Ready for review.

**Pull request merged**